tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -31,8 +32,36 @@ limitations under the License.
 #include "tensorflow/lite/schema/schema_generated.h"
 #include "tensorflow/lite/schema/schema_utils.h"
 
+//#include "stdio.h"
 namespace tflite {
+template <class T>
+std::vector<int> FlatBufferIntArrayToVector(T* flat_array) {
```

Comment:
We try to avoid any use of std::vectors or other STL containers, because they usually bring in a lot of code and require dynamic memory allocation. Can you talk a bit about why this is needed in this case?

Reply:
DO you have any substitute to this function call or its implementation   tflite::FlatBufferIntArrayToVector(op->intermediates())